server/README.md

```diff
@@ -56,55 +56,157 @@ then create it from the correct `.env` file (for example, on the Sensors Admin s
+
+      - Version number = <version number>
+
+      - Title = v<version number> <envrionment>
+
+      - Description - <leave blank>
+
+      - Drag and drop or upload the generated `.bin` file
+
+1. Go to Devices
+
+   1. open a device that is online and click "Edit"
 
-1. If you determined that the database changes require downtime:
+   1. In the firmware dropdown, select the newly uploaded firmware, check the box "Flash now", and click "Save"
 
-   1. Send a notification to all the responder phones informing them of the downtime
+1. Go back to Firmware
+
+   1. Hover over your new version, click on "Release firmware", fill in the following, and click "Next"
+
+      1. Release target = "Product default" to deploy to all Beta Borons
+
+      1. Select "Intelligent"
+
+   1. Read, verify, check the box, and click "Release this firmware"
+
+1. Trigger a test alert using a real BetaTest Boron device
+
+## 6. Update production server
+
+1. If you determined that this deployment's changes require downtime:
+
+   1. Send a notification to all the responder phones informing them of the downtime. For example: "Notice: Your Brave Sensor System is down for maintenance. During this time, you may not receive bathroom alerts. You will receive another text message when everything is back online. Thank you for your patience. Have a nice day!"
 
    1. Take the production server offline `helm uninstall production`
 
 1. Update the production DB and secrets, as necessary
 
-1. Run the command `helm upgrade production ~/BraveSensor/server/sensor-helm-chart`
+1. If the deployment's changes did _not_ require downtime,
+
+   1. run the command `helm upgrade production ~/BraveSensor/server/sensor-helm-chart`.
+
+   Otherwise
+
+   1. run the command `helm install production ~/BraveSensor/server/sensor-helm-chart`.
+
+   1. Send a notification to all the responder phones informing them that everything is back online. For example: "Notice: Your Brave Sensor System is now back online and functioning normally. Thank you!"
+
+## 7. Update production firmware
+
+1. on your local machine, open Visual Studio Code in the `firmware/boron-ins-fsm` directory and open the Particle Workbench extension
 
-1. If you determined that the database changes require downtime:
+1. run `git checkout production && git pull origin production` to get the latest version of the the code
 
-   1. Send a notification to all the responder phones informing them that everything is back online
+1. in `BraveSensorProductionFirmware.ino`, change `BRAVE_PRODUCT_ID` to the ID of the "Production Sensor Devices" project (`15479`) [NOTE: do NOT commit this change]
+
+1. click Target --> "Configure for device" and type `2.0.0` to choose the OS and then `boron` to choose the device
+
+1. click Compile --> "Local Compile"
+
+1. copy the generated `/target/2.0.0/boron/boron-ins-fsm.bin` file somewhere to keep for reference. Rename to `v<version number>\_production.bin`
```

Comment:
Again, providing an example of the renamed bin file might be useful, for example `v50001_production.bin`

Reply:
Done!